### PR TITLE
Added engaging config

### PR DESCRIPTION
**Pull request by PhilPalmer**
---
name: engaging
about: A new cluster config for the [MIT Engaing HPC Cluster](https://engaging-web.mit.edu/eofe-wiki/).
---

Please follow these steps before submitting your PR:

- [x] If your PR is a work in progress, include `[WIP]` in its title
- [x] Your PR targets the `master` branch
- [x] You've included links to relevant issues, if any

Steps for adding a new config profile:

- [x] Add your custom config file to the `conf/` directory
- [x] Add your documentation file to the `docs/` directory
- [x] Add your custom profile to the `nfcore_custom.config` file in the top-level directory
- [x] Add your custom profile to the `README.md` file in the top-level directory
- [x] Add your profile name to the `profile:` scope in `.github/workflows/main.yml`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -219,6 +219,7 @@ Currently documentation is available for the following pipelines within specific
   - [EVA](docs/pipeline/eager/eva.md)
 - mag
   - [EVA](docs/pipeline/mag/eva.md)
+  - [Engaging](docs/pipeline/mag/engaging.md)
```
Comment (maxulysse) [inline README.md:222]: `Engaging` should be before `EVA`

File: pipeline/mag.config [hunk 2]
```diff
@@ -10,4 +10,5 @@
 
 profiles {
   eva { includeConfig "${params.custom_config_base}/conf/pipeline/mag/eva.config" }
+  engaging { includeConfig "${params.custom_config_base}/conf/pipeline/mag/engaging.config" }
```
Comment (maxulysse) [inline pipeline/mag.config:13]: `engaging` should be before `eva`

**Review by maxulysse — COMMENTED** [review]
I do prefer list ordered, but apart from that, looks good

**Comment by PhilPalmer** [comment]
> I do prefer list ordered, but apart from that, looks good

Thanks @maxulysse for the PR review and feedback! I've addressed your comments :)

**Review by maxulysse — APPROVED** [review]

**Pull request merged**